pkg/revocation/db_test.go

```diff
@@ -151,3 +151,48 @@ func TestRevocationDB_Put_error(t *testing.T) {
 		}
 	})
 }
+
+func TestRevocationDB_List(t *testing.T) {
+	ctx := testcontext.New(t)
+	defer ctx.Cleanup()
+
+	testrevocation.RunDBs(t, func(t *testing.T, revDB extensions.RevocationDB, db storage.KeyValueStore) {
+		keys, chain, err := testpeertls.NewCertChain(2, storj.LatestIDVersion().Number)
+		require.NoError(t, err)
+		keys2, chain2, err := testpeertls.NewCertChain(2, storj.LatestIDVersion().Number)
+		require.NoError(t, err)
+
+		// test list no revocations, should not error
+		revs, err := revDB.List(ctx)
+		assert.NoError(t, err)
+		assert.Nil(t, revs)
+
+		// list 1,2 revocations
+		firstRevocation, err := extensions.NewRevocationExt(keys[peertls.CAIndex], chain[peertls.LeafIndex])
+		require.NoError(t, err)
+
+		err = revDB.Put(ctx, chain, firstRevocation)
+		require.NoError(t, err)
+		revs, err = revDB.List(ctx)
+		assert.NoError(t, err)
+		assert.Equal(t, 1, len(revs))
```

Comment:
Might want to `require` here as well. If `len(revs)` is actually 0, and execution continues, you'll hit a panic on the next line (which will hide the assertion error that was added on this line, causing confusion).

It might even be a good idea to use `require` exclusively in this file. The `assert` package is only helpful when you are sure nothing will crash if execution carries on after an assertion failure, and when there is reason to think it's worthwhile for execution to carry on.